fxa/core.py

```diff
@@ -278,12 +285,7 @@ def get_email_status(self):
         return resp
 
     def verify_email_code(self, code):
-        body = {
-            "uid": self.uid,
-            "code": code,
-        }
-        url = "/recovery_email/verify_code"
-        self.apiclient.post(url, body)  # note: not authenticated
+        return self.client.verify_email_code(self.uid, code) # note: not authenticated
```

Comment:
You do not need to push force you can just add new commits in your branch and it will update the PR.
It is better to update a PR rather than creating a new one because we can follow the discussions and people taking part on the discussion get notified.